test/typescript/init.test.ts

```diff
@@ -205,6 +205,12 @@ const v: string = i18next.t('my.key');
 const de = i18next.getFixedT('de');
 const z: string = de('myKey');
 
+// verify we can get the German data
+const data = i18next.getDataByLanguage('de');
+if (data.translation.myKey !== de('myKey')) {
```

Comment:
This is type usage test only, so we only need the line 209, please remove the `if` block to keep these tests as brief as possible.

Reply:
updated with a change - please look :)